naps/soc/platform/zynq/zynq_soc_platform.py

```diff
@@ -64,3 +63,7 @@ def pack_bitstream_fatbitstream(self, name: str, build_products: BuildProducts):
 
     def program_fatbitstream(self, *args, **kwargs):
         program_fatbitstream_ssh(self, *args, **kwargs)
+
+    def pydriver_memory_accessor(self, memorymap):
+        contents = (Path(__file__).parent / "memory_accessor_devmem.py").read_text()
+        return contents.replace("@@SIZE@@", str(memorymap.byte_len))
```

Comment:
Instead of textually replacing in the source code, maybe this could be something like `return contents + f"\nMemoryAccessor = DevMemAccessor({memorymap.byte_len})\n`

Reply:
I am not sure if I like textual replacement or building python code from python better, but sure.

Reply:
I agree, that both options have a crtain smell; dont have any real preference here